spiffworkflow-frontend/src/components/DataStoreListTiles.tsx

```diff
@@ -41,10 +41,9 @@ export default function DataStoreListTiles({
       if (defaultDataStores) {
         setDataStores(defaultDataStores);
       } else {
-        let queryParams = '?per_page=1000';
-        if (processGroup) {
-          queryParams = `${queryParams}&process_group_identifier=${processGroup.id}`;
-        }
+        const queryParams = `?per_page=1000&process_group_identifier=${
```

Comment:
maybe if you have a quick minute to omit `process_group_identifier` completely when id is not there, otherwise can merge this.

Reply:
Sadly it needs to be there, at least until I port the typeahead data store to have a location, since we still have to support requests coming in with no location vs the "" empty top level location...